exonum_client/protobuf_loader.py

```diff
@@ -142,10 +147,13 @@ def initialize(self) -> None:
         # Add a directory with exonum_modules into the Python path:
         sys.path.append(self._proto_dir)
 
+        logging.debug("Successfully initialized ProtobufLoader.")
```

Comment:
Again, the context info is required: for which client it was created?

And I think that it worth creating a log entry when a reference counter is increased/decreased, so later we can see in logs if we forgot to deinitialize some loader before creating a new one.